examples/pyaudio_oscope_local.py

```diff
@@ -54,3 +54,7 @@
     import sys
     if sys.flags.interactive == 0:
         app.exec_()
+
+# stop and close pyaudio node to properly close pyaudio
```

Comment:
I've tried registering cleanup code in Pyaudio-__init__
But I'm launching the example from Spyder IDE and it looks like cleanup code only gets called when you restart the kernel. 
There are also issues whith using instance methods as cleanup code: https://stackoverflow.com/questions/16333054/what-are-the-implications-of-registering-an-instance-method-with-atexit-in-pytho
Anyway, the stop and close way should be an option and not produce errors (the ones that the other proposed changes fix)